Plan/common/src/main/resources/assets/plan/locale/locale_FR.txt

```diff
@@ -1,336 +1,336 @@
-Health - Active Playtime Comparison Decrease    ||  Active players might be running out of things to do (Played ${0} vs ${1}, last two weeks vs weeks 2-4)
-Health - Active Playtime Comparison Increase    ||  Active players seem to have things to do (Played ${0} vs ${1}, last two weeks vs weeks 2-4)
-Health - Downtime                               ||  Total Server downtime (No Data) was ${0}
-Health - New Player Join Players, No            ||  New Players may not have players to play with when they join (${0} on average)
-Health - New Player Join Players, Yes           ||  New Players have players to play with when they join (${0} on average)
-Health - New Player Stickiness                  ||  ${0} of new players have stuck around (${1}/${2})
-Health - No Servers Inaccuracy                  ||  No Bukkit/Sponge servers to gather session data - These measures are inaccurate.
-Health - Player Play on Network                 ||  players played on the network:
-Health - Player Register Server                 ||  players register on servers per day/server on average.
-Health - Player Visit Server                    ||  players visit on servers per day/server on average.
-Health - Regular Activity Change                ||  Number of regular players has
-Health - Regular Activity Change Decrease       || decreased (${0})
-Health - Regular Activity Change Increase       || increased (+${0})
-Health - Regular Activity Change Zero           || stayed the same (+${0})
-Health - Regular Activity Remain                ||  ${0} of regular players have remained active (${1}/${2})
-Health - Single Servers Inaccuracy              ||  Single Bukkit/Sponge server to gather session data.
-Health - TPS Above Low Threshold                ||  Average TPS was above Low Threshold ${0} of the time
-Health - TPS Low Dips                           ||  Average TPS dropped below Low Threshold (${0}) ${1} times
-HTML - FREE_DISK_SPACE                          || Free Disk Space
-HTML - DISK_SPACE                               || DISK SPACE
\ No newline at end of file
+WebServer - Notify HTTP                         || Serveur Web : Aucun certificat -> Utilisation du serveur HTTP pour la visualisation.
+WebServer - Notify HTTP User Auth               || Serveur Web : Authentification utilisateur désactivée ! (Non sécurisée avec HTTP)
+WebServer - Notify no Cert file                 || Serveur Web : Fichier KeyStore du certificat introuvable : ${0}
+WebServer FAIL - Port Bind                      || Le Serveur Web n'a pas été initialisé avec succès. Le port (${0}) est-il actuellement utilisé ?
+WebServer FAIL - SSL Context                    || Serveur Web : Échec d'initialisation du contexte SSL.
+WebServer FAIL - Store Load                     || Serveur Web : Échec du chargement du certificat SSL.
+Yesterday                                       || 'Hier'
+Today                                           || 'Aujourd"hui'
```

Comment:
I'll be able to correct the word "Aujourd'hui" when you'll corrected the bug we talked about on Discord. For the moment ' is replaced by " and it's perfectly working ;)